manifests/profile/quod/perl.pp

```diff
@@ -0,0 +1,284 @@
+    'libmarc-record-perl',
+    'libmath-round-perl',
+    'libmime-tools-perl',
+    'libmime-types-perl',
+    'libmodule-implementation-perl',
+    'libmodule-pluggable-perl',
+    'libmodule-runtime-conflicts-perl',
+    'libmodule-runtime-perl',
+    'libmoo-perl',
+    'libmoose-perl',
+    'libmoox-types-mooselike-perl',
+    'libmouse-perl',
+    'libmp3-info-perl',
+    'libmro-compat-perl',
+    'libmultidimensional-perl',
+    'libnamespace-autoclean-perl',
+    'libnamespace-clean-perl',
+    'libnet-daemon-perl',
+    'libnet-dns-perl',
+    'libnet-domain-tld-perl',
+    'libnet-http-perl',
+    'libnet-idn-encode-perl',
+    'libnet-ip-perl',
+    'libnet-ldap-perl',
+    'libnet-smtp-ssl-perl',
+    'libnet-ssleay-perl',
+    'libnet-z3950-zoom-perl',
+    'libnumber-compare-perl',
+    'libole-storage-lite-perl',
+    'libossp-uuid-perl',
+    'libpackage-deprecationmanager-perl',
+    'libpackage-stash-perl',
+    'libpackage-stash-xs-perl',
+    'libpadwalker-perl',
+    'libparams-classify-perl',
+    'libparams-util-perl',
+    'libparams-validate-perl',
+    'libparams-validationcompiler-perl',
+    'libparse-debianchangelog-perl',
+    'libparse-recdescent-perl',
+    'libperl4-corelibs-perl',
+    'libreadonly-perl',
+    'libreadonly-xs-perl',
+    'libregexp-assemble-perl',
+    'librole-tiny-perl',
+    'libscalar-list-utils-perl',
+    'libsocket-perl',
+    'libsocket6-perl',
+    'libsort-naturally-perl',
+    'libspecio-perl',
+    'libspreadsheet-writeexcel-perl',
+    'libstrictures-perl',
+    'libsub-exporter-perl',
+    'libsub-exporter-progressive-perl',
+    'libsub-identify-perl',
+    'libsub-install-perl',
+    'libsub-name-perl',
+    'libswitch-perl',
+    'libsys-hostname-long-perl',
+    'libtask-weaken-perl',
+    'libtemplate-perl',
+    'libterm-readkey-perl',
+    'libtest-fatal-perl',
+    'libtest-nowarnings-perl',
+    'libtest-requires-perl',
+    'libtext-charwidth-perl',
+    'libtext-csv-xs-perl',
+    'libtext-glob-perl',
+    'libtext-iconv-perl',
+    'libtext-levenshtein-perl',
+    'libtext-unidecode-perl',
+    'libtext-wrapi18n-perl',
+    'libthrowable-perl',
+    'libtie-ixhash-perl',
+    'libtimedate-perl',
+    'libtry-tiny-perl',
+    'libtypes-serialiser-perl',
+    'libunicode-string-perl',
+    'libuniversal-require-perl',
+    'liburi-perl',
+    'libuuid-perl',
+    'libuuid-tiny-perl',
+    'libvariable-magic-perl',
+    'libwww-curl-perl',
+    'libwww-perl',
+    'libwww-robotrules-perl',
+    'libxml-libxml-perl',
+    'libxml-libxslt-perl',
+    'libxml-namespacesupport-perl',
+    'libxml-parser-perl',
+    'libxml-sax-base-perl',
+    'libxml-sax-expat-perl',
+    'libxml-sax-perl',
+    'libxml-simple-perl',
+    'libxml-twig-perl',
+    'libxml-writer-perl',
+    'libxml-xpath-perl',
+    'libxml-xpathengine-perl',
+    'libyaml-appconfig-perl',
+    'libyaml-libyaml-perl',
+    'libyaml-perl',
+    'libyaml-syck-perl',
+    'xmlformat-perl']:
```

Comment:
Where did this list come from? Why is it different from "dependencies"?

Reply:
These are perl packages. Dependencies are prerequisites for these packages as well as for anything else that needs to be installed.

The layout is based on what we have in www_lib.

Reply:
And example would be the oracle auth which apparently is a pre-requisite for some perl packages from what I gathered.